src/linear_algebra/dual.lean

```diff
@@ -532,13 +512,13 @@ noncomputable def quot_annihilator_equiv (W : subspace K V) :
   the dual of `V`. -/
 def dual (W : subspace K V) : subspace K (module.dual K V) :=
 { carrier := { φ | ∃ ψ : module.dual K W, φ = W.dual_lift ψ },
```

Comment:
This is `dual_lift.range`

Reply:
I've replaced all instances of `subspace.dual` with `subspace.dual_lift.range` now. Some of the dot notation is getting a bit long, e.g. `W.dual_lift.range.quotient`, but I think that is fine.